### ISSUE TITLE
Adding documentation on how to add your own source

### ISSUE THREAD

**Issue by LouayHanouche** [Layendan/NineAnimator-Tauri#40]
i just found this app and i like it , but i cant figure out how to add my own source and i couldn't find any documentation on this problem

**Comment by Layendan** [comment]
Yeah, I was working on updating how the app handles sources. Because how sources worked kept changing, I didn't bother writing documentation. But it is something that I can work on.

**Comment by LouayHanouche** [comment]
Thank you its going to help a lot since it will be possible to add multiple sources in case one failed or don't have subtitles per example .
